.github/workflows/test-integration.yml

```diff
@@ -2,7 +2,7 @@ name: test-integration
 on:
   pull_request:
   push:
-    branches: [ master, develop ]
+    branches: [ master, develop, feature/self-asserted-tokens ]
```

Comment:
You know that instead of keeping a list, you can use wildcards here? i.e. `feature/*`

Reply:
Good suggestion!